src/clib/lib/config/config_schema_item.cpp

```diff
@@ -409,19 +409,6 @@ bool config_schema_item_validate_set(const config_schema_item_type *item,
 
                         free(relocated);
 
-                        if (strstr(value, "/")) {
-                            /*
-                             * value not naked name, which means it was relative path
-                             * that wasn't found
-                             */
-
-                            error_list.push_back(std::string(
```

Comment:
Looking at the history, I may have messed this message up. I think perhaps that the code works in such a way that if value contains "/" then the executable does not exist. If I follow the logic then it is supposedly a relative path that wasn't found. Is that still covered?

Reply:
If there is a relative path that is not found or not fully resolved yet when the check is running then I guess an abort will happen. 
```
                        /*
                         * res_env_alloc_PATH_executable aborts if some parts of the path is
                         * not an existing dir, so call it only when its an absolute path
                         */
                        char *path_exe = res_env_alloc_PATH_executable(value);
```

Reply:
Can we somehow provoke such an abort in a test?

Reply:
I am struggling a bit to do it but the execution logic for this piece of code is a bit complex and it is taking some time